src/main/java/com/thoughtworks/gauge/command/StartCommand.java

```diff
@@ -42,7 +44,11 @@ public void execute() throws Exception {
         Server server;
         MessageProcessorFactory messageProcessorFactory = new MessageProcessorFactory(staticScanner);
         RunnerServiceHandler runnerServiceHandler = new RunnerServiceHandler(messageProcessorFactory, multithreading, numberOfStreams);
-        server = ServerBuilder.forPort(0).addService(runnerServiceHandler).executor((Executor) Runnable::run).build();
+        server = NettyServerBuilder
+            .forAddress(new InetSocketAddress(LOCALHOST, 0))
+            .addService(runnerServiceHandler)
+            .executor((Executor) Runnable::run)
+            .build();
```

Comment:
I noticed this on their API docs. https://github.com/grpc/grpc-java/issues/1784 
Still worth the change?

Reply:
This comment seems to suggest this approach for the exact same problem: https://github.com/grpc/grpc-java/issues/5791#issuecomment-496289057